packages/@react-spectrum/menu/src/ActionMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import {ActionButton} from '@react-spectrum/button';
+import {filterDOMProps} from '@react-aria/utils';
+import {FocusableRef} from '@react-types/shared';
+// @ts-ignore
+import intlMessages from '../intl/*.json';
+import {Menu} from './Menu';
+import {MenuTrigger} from './MenuTrigger';
+import More from '@spectrum-icons/workflow/More';
+import React from 'react';
+import {SpectrumActionMenuProps} from '@react-types/menu';
+import {useMessageFormatter} from '@react-aria/i18n';
+
+function ActionMenu<T extends object>(props: SpectrumActionMenuProps<T>, ref: FocusableRef<HTMLButtonElement>) {
+  let formatMessage = useMessageFormatter(intlMessages);
+  let buttonProps = filterDOMProps(props);
+  if (buttonProps['aria-label'] === undefined) {
```

Comment:
this will always be undefined, we need to add labellable to `filterDOMProps` https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/utils/src/filterDOMProps.ts#L20
https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/utils/src/filterDOMProps.ts#L46

Reply:
fixed - i did not spot the `labelable` check in that `filterDOMProps`